### ISSUE TITLE
Search function & CRT shaders

### ISSUE THREAD

**Issue by SuperMantis** [webrcade/webrcade#22]
Hi,

First of all, big props to the team for creating this marvellous project. 
I'm extremely excited to see what the future holds and look forward to experience the progress.

The two functions I'd really love to see implemented are the following:

1. Search Function. A simple game search function so you don't have to scroll through the whole game library each time you want to play a specific game.

2. CRT shaders. Would love to see an option to apply a simple CRT shader or scanlines so the games will look better when playing them on bigger screens or on TVs. A lightweight shader would be ideal so it's not so taxing on the performance on lower end devices.

Keep up the awesome work everyone, cheers!

**Comment by raz0red** [comment]
Agreed, definitely want to add some simple shaders.
Search is also something that is absolutely needed.

**Comment by sysconf16** [comment]
I'd also like to see a search function added.

**Comment by Casuallynoted** [comment]
I agree that a search function is also sorely needed. The second your game library gets pretty comprehensive, it's really difficult to navigate.

**Comment by raz0red** [comment]
Search and shaders are available now (0.2.1)
